### PR TITLE
Remove remaining bazel references

### PR DESCRIPTION
**Pull request by rifelpet**

### CODE REVIEW

**Review by olemarkus — APPROVED** [review]

File: docs/contributing/testing.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ kubetest2 kops \
   --test \
   --cloud-provider=aws \
   --cluster-name=my.testcluster.com \
-  --kops-binary-path=${KOPS_ROOT}/.bazel-bin/cmd/kops/linux-amd64/kops \
+  --kops-binary-path=${KOPS_ROOT}/.build/dist/$(go env GOOS)/$(go env GOARCH)/kops \
```
Comment (hakman) [inline docs/contributing/testing.md:52]: Not quite sure GOOS and GOARCH should be used here. Most likely the kOps binary should be `linux/amd64`.

Comment (hakman) [reply docs/contributing/testing.md:52]: Yes, but this assumes the test cluster GOOS & GOARCH would be the same as local one.

**Pull request merged**